src/h5web/visualizations/line/DataGlyphs.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useLoader } from 'react-three-fiber';
+import { TextureLoader, BufferGeometry } from 'three';
+import { Glyph } from './models';
+import circleURL from './sprites/circle.png';
+import crossURL from './sprites/cross.png';
+import squareURL from './sprites/square.png';
+
+const GLYPH_URLS = {
+  [Glyph.Circle]: circleURL,
+  [Glyph.Cross]: crossURL,
+  [Glyph.Square]: squareURL,
+};
+
+interface Props {
+  geometry: BufferGeometry;
+  color: string;
+}
+
+function DataGlyphs(props: Props): JSX.Element {
+  const { geometry, color } = props;
+  const sprite = useLoader(TextureLoader, GLYPH_URLS[Glyph.Cross]);
```

Comment:
Thinking ahead, I think the glyph URL should be a prop, which defaults to `GLYPH_URLS[Glyph.Cross]`. This way, people can reuse it and pass their own sprite URL.

The enum for the type of glyph does not make much sense anymore. Perhaps we should change it to `GlyphUrls` directly (instead of having the URLs in a separate object). The way I see, we'd then export the enum so people can use it to pass a built-in sprite URL to `DataGlyphs` (or to a parent component that would forward the prop).